allauth/socialaccount/providers/authentiq/provider.py

```diff
@@ -0,0 +1,94 @@
+from allauth.account.models import EmailAddress
+from allauth.socialaccount.app_settings import \
+    QUERY_EMAIL, EMAIL_REQUIRED, EMAIL_VERIFICATION
+from allauth.socialaccount.providers.base import AuthAction, ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class Scope(object):
+    NAME = 'aq:name'
+    EMAIL = 'email'
+    PHONE = 'phone'
+    EMAIL_VERIFIED = 'email~rs'
+    PHONE_VERIFIED = 'phone~rs'
+    ADDRESS = 'address'
+    PUSH = 'aq:push'
+
+
+IDENTITY_CLAIMS = frozenset([
+    'sub', 'name', 'given_name', 'family_name', 'middle_name',
+    'nickname', 'preferred_username', 'profile', 'picture',
+    'website', 'email', 'email_verified', 'gender', 'birthdate',
+    'zoneinfo', 'locale', 'phone_number', 'phone_number_verified',
+    'address', 'updated_at'
+])
+
```

Comment:
If it does not get dropped from the IDENTITY_CLAIMS set then it will probably be in extras indeed.

IIRC on Omniauth we add it in the normal profile data and leave other things to extras. Specifically

- aq:name for name, providing :name, :first_name, :last_name and additionally :middle_name will be available in :extra
- email providing :email and additionally :email_verified will be available in :extra
- phone providing :phone and additionally :phone_type and :phone_number_verified will be available in :extra
- address providing :location with the following format:
- aq:location providing :geolocation (geo coordinates and address from a reverse lookup) with the following format:
- aq:push to request permission to sign in via Push Notifications in the Authentiq ID app
- :locale and :zoneinfo will be available in :extra regardless of the requested scopes. The format of these strings is:
    -  locale providing :locale in the language_territory format
    -  zoneinfo providing :zoneinfo in the Continent/City format


Even more info, here https://github.com/AuthentiqID/omniauth-authentiq/wiki/Scopes,-callback-url-configuration-and-responses

Proceed as you believe better. It surely is not a deal breaker.